app/static/settings/languages.json

```diff
@@ -45,5 +46,7 @@
   {"name": "Thai (ไทย)", "value": "lang_th"},
   {"name": "Turkish (Türk)", "value": "lang_tr"},
   {"name": "Ukranian (Український)", "value": "lang_uk"},
-  {"name": "Vietnamese (Tiếng Việt)", "value": "lang_vi"}
+  {"name": "Vietnamese (Tiếng Việt)", "value": "lang_vi"},
+  {"name": "Xhosa (isiXhosa)", "value": "lang_xh"},
+  {"name": "Zulu (isiZulu)", "value": "lang_zu"}
```

Comment:
Are these languages (Xhosa and Zulu) supported by Google? I tried enabling them in the config, but it looks like the results are still in English, which would indicate that Google does not support these language values.

Reply:
https://www.google.co.za/?hl=xh
https://www.google.com/?hl=zu

Google translate also support for that languages